pkg/products/solutionexplorer/reconciler.go

```diff
@@ -217,6 +238,31 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
 
+func (r *Reconciler) reconcileConsoleLink(ctx context.Context, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+	cl := &consolev1.ConsoleLink{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "rhmi-solution-explorer",
+		},
+		Spec: consolev1.ConsoleLinkSpec{
+			ApplicationMenu: &consolev1.ApplicationMenuSpec{},
+		},
+	}
+
+	_, err := controllerutil.CreateOrUpdate(ctx, serverClient, cl, func() error {
+		cl.Spec.ApplicationMenu.ImageURL = "https://avatars1.githubusercontent.com/u/41683098"
+		cl.Spec.ApplicationMenu.Section = "Red Hat Managed Integrations"
```

Comment:
@philbrookes Can the Solution Explorer be added to the current "Red Hat Applications" section instead of being included in a new one? This is the recommended location based on the UX guidelines for using this application menu as we only have one item and it should go with any other Red Hat Applications such as OCM and CRW.

Reply:
Thanks @tiffanynolan I've made that change now

Reply:
Great! Thanks, @philbrookes